site/en/docs/webstore/best_practices/index.md

```diff
@@ -9,10 +9,33 @@ description: >
 This page has advice on how you should implement your app and list it in the store. As the store
 matures and we learn from developers' experiences, these recommendations will be updated.
 
-## Design a great app
+## Design a high-quality extension
+
+We encourage you to develop extensions that are of high quality. High-quality extensions conform to
+standards of performance, security, and user experience, as summarized by the following guidelines:
+
+* **Compliance:** Does the extension comply with our [program policies]? Extensions must not violate
+any of these policies.
+
+* **Manifest V3:** Is the extension built on Manifest V3? Manifest V3 reflects the aims of our
+  platform vision, and all High Quality extensions should use it. (See this
+  [MV3 overview][mv3-overview].)
+
+* **Security:** Is the extension safe for users? Make sure your extension does not pose security
+  threats, does not use deceptive installation tactics and meets Chrome Web Store's data privacy
+  requirements (see this [FAQ][user-data-faq]).
+
+* **Performance:** Does the extension function at an outstanding level? High-quality extensions
+  don't just perform their intended action, but perform at an extremely high level. 
+
+* **User experience:** Is the extension a joy to use? Make sure your extensions are easy to
+  install and manage, with a good-looking, intuitive and seamless user experience.
+
+* **Store listing:** Does the store listing page provide quality and well-designed images (icon,
+  tile, marquee, and screenshots)? Images should not be blurry or too busy. Additionally, the store
+  listing page should be complete with all fields filled out (see [Supplying images] [supplying-images]
+  and [Completing your store listing] [completing-listing] for more listing tips).
```

Comment:
```suggestion
* **Store listing:** The extension's [Chrome Web Store listing][completing-listing] should sets the
  user's expectations and clearly communicates what the extension does.   All image assets (icon,
  tile, marquee, and screenshots) [should be provided][supplying-images]. Images should not be
  blurry or too busy. [Privacy information][dashboard-privacy] (permissions justifications, the extension's privacy policy, data use disclosures, etc.) must be accurate and up to date.
```

Note that I have another suggestion to add the dashboard-privacy link to the footer further down in this review.

The spaces between the link text and footer ID are breaking the links.